src/K9OCRS/ClientApp/src/areas/applications/Catalog/Classes.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import * as classTypesClient from '../../../util/apiClients/classTypes';
+import { Container, CardImg, Spinner, Row, Col } from "reactstrap";
+import Table from "../../../shared/components/Table";
+import sectionColumns from './components/sectionColumns';
+import './components/style.scss';
+
+const Classes = props => {
+    const { classTypeId } = useParams();
+    const [classDetail, setClassDetail] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [alerts, setAlerts] = useState([]);
+    
+    // const description = location.state.description;
+    // const requirements = location.state.requirements;
+
+    useEffect(() => {
+        async function getTest() {
+          if (classTypeId) {
+            try {
+              const res = await classTypesClient.getClassTypeByID(classTypeId);
+              setClassDetail(res?.data);
+              setLoading(false);
+            } catch(err) {
+              setLoading(false);
+              setAlerts([{ color: 'danger', message: 'We\'re having issues getting the details for this class' }]);
+            }
+          }
+      
+        }
+        getTest();
+      }, [classTypeId]);
+
+      const photoArr = classDetail.photos;
+
+
+    return (
+        <>
+        <h2>{classDetail.title} </h2>
+        
+        <p>Class Catalog / {classDetail.title}</p>
+        <Container className="px-sm-5 container-sm" fluid>
+        <CardImg
+          className="pb-4 heroImg"
+          alt={`Image for the ${classDetail.title} class`}
+          src={classDetail.imageUrl}
+          top
+        />
```

Comment:
Don't use CardImg components for every kind of image, it's supposed to be used with react-strap's Card component. Use a good ol' img tag instead. Also, the linter will find an "alt" attribute with the word "Image" in it redundant.
```suggestion
        <img
          className="pb-4 heroImg"
          alt={`The ${classDetail.title} class`}
          src={classDetail.imageUrl}
          top
        />
```